### PR TITLE
Added import of additional cloud-init variables

### PR DESCRIPTION
**Pull request by bretttegartms**
Added ability to import k:v pairs on disk to instance cloud-init data.

### CODE REVIEW

File: lib/python/treadmill_aws/api/host.py [hunk 1]
```diff
@@ -35,7 +35,8 @@ def create_hosts(ami,
                 secgroup_ids=secgroup,
                 instance_type=size,
                 subnet_id=subnet,
-                disk=disk)
+                disk=disk,
+                instance_vars={})
```
Comment (andreikeis) [inline lib/python/treadmill_aws/api/host.py]: instance_vars=None)

if not instance_vars:
  instance_vars = {}

File: lib/python/treadmill_aws/cli/admin/aws/instance.py [hunk 2]
```diff
@@ -102,10 +105,16 @@ def configure(instance):
         help='Root parition size, e.g. 100G',
         callback=aws_cli.convert_disk_size_to_int
     )
+    @click.option(
+        '--data',
+        required=False,
+        help='Instance data, in YAML',
+        type=click.Path(exists=True, readable=True)
```
Comment (andreikeis) [inline lib/python/treadmill_aws/cli/admin/aws/instance.py]: I would make it file, not path, then you can pass it in stdin:

--data - ... < data.yml

File: lib/python/treadmill_aws/api/host.py (hunk 1 above)
Comment (bretttegartms) [reply lib/python/treadmill_aws/api/host.py]: OK, changed to None.

File: lib/python/treadmill_aws/cli/admin/aws/instance.py (hunk 2 above)
Comment (bretttegartms) [reply lib/python/treadmill_aws/cli/admin/aws/instance.py]: OK, changed to click.File

**Pull request merged**